### PR TITLE
Exponential function documentation.

### PR DESCRIPTION
**Pull request by EVANSTATS**
Hello @lilyclements, have a look at this.

### CODE REVIEW

**Comment by EVANSTATS** [comment]
I shouldn't have added changes to the DESCRIPTION file here. Kindly check it also. Am not sure if it was necessary but I kept getting a warning message. We can probably discuss this during a call.

File: DESCRIPTION [hunk 1]
```diff
@@ -10,4 +10,6 @@ Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.2.3
 Imports: 
-    magrittr
+    magrittr,
+Remotes:
+    pacificclimate/climdex.pcic
```
Comment (lilyclements) [inline DESCRIPTION:16]: @EVANSTATS nice! Good memory, and very observant!

Comment (lilyclements) [reply DESCRIPTION:16]: @EVANSTATS only thing is that we want to have `climdex.pcic` in `Imports` too -

that is,

```
Imports: 
    climdex.pcic,
    magrittr,
Remotes:
    pacificclimate/climdex.pcic
```

**Review by lilyclements — CHANGES_REQUESTED** [review]
Minor comment, but very good stuff!

File: DESCRIPTION (hunk 1 above)
Comment (EVANSTATS) [reply DESCRIPTION:16]: Aaah thank you.

**Review by lilyclements — APPROVED** [review]

**Pull request merged**